packages/docs/build/pug-to-json.js

```diff
@@ -0,0 +1,80 @@
+"use strict"
+const pug = require('pug')
+const walk = require('pug-walk')
+const compiler = require('vue-template-compiler')
+
+module.exports = function (content) {
+  this.cacheable()
+
+  const page = { children: [] }
+  let pageSection
+
+  function startSection () {
+    pageSection = {
+      type: 'section',
+      children: [],
+    }
+    page.children.push(pageSection)
+  }
+
+  pug.render(content, {
+    compileDebug: false,
+    filename: this.resourcePath,
+    filters: {
+      code: text => text
+    },
+    plugins: [{
+      preFilters (ast) {
+        const parents = []
+        walk(ast, node => {
+          if (!pageSection || ['h2', 'up-next', 'usage', 'usage-new', 'api', 'examples'].includes(node.name)) {
```

Comment:
Not sure about this bit, it seems like most components except markup and alerts are in a new section. Might be a better idea to be explicit about this?